app/classifier/feedback/helpers.js

```diff
@@ -16,4 +16,8 @@ const isThereFeedback = (subject, workflow) =>
     return newResult;
   }, false);
 
-export { isFeedbackActive, isThereFeedback };
+const renderUniqueFeedback = (annotations, feedback) => {
+  const annotationFrame = annotations.map(item => item.value.map(value => value.frame));
```

Comment:
`Array.map` maps one array onto another array, so either name this `annotationFrames` or reduce the array down to a single frame value.